apps/nextjs-website/src/components/atoms/TabComponent/TabComponent.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+import React, { FC, ReactNode } from 'react';
+import { Box, Tab, Tabs, Typography } from '@mui/material';
+
+export type TabPanelProps = {
+  readonly children?: ReactNode;
+  readonly index: number;
+  readonly value: number;
+};
+
+const TabPanel: FC<TabPanelProps> = ({ children, value, index, ...other }) => {
```

Comment:
I think `other` doesn't contain anything, am I wrong?

Reply:
Yes, it was there to optionally pass some style or classes, but we can add it later if we need it